src/bentoml/bentos.py

```diff
@@ -238,14 +239,13 @@ def push(
     *,
     force: bool = False,
     _bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store],
+    _yatai_client: YataiClient = Provide[BentoMLContainer.yatai_client],
 ):
     """Push Bento to a yatai server."""
-    from bentoml._internal.yatai_client import yatai_client
-
     bento = _bento_store.get(tag)
     if not bento:
         raise BentoMLException(f"Bento {tag} not found in local store")
-    yatai_client.push_bento(bento, force=force)
+    _yatai_client.push_bento(bento, force=force)
```

Comment:
can we refactor this change out to a different PR?

Reply:
This is necessary for the unit tests

Reply:
oh I mean the change for adding the yatai_client to `BentoMLContainer`.

Reply:
yes, this refactor makes it easier to test the REMOTE pulling serialization strategy